CMakeLists.txt

```
@@ -52,7 +52,7 @@ OPTION(TAGS				"Generate tags"						OFF)
 OPTION(PROFILE				"Generate profiling information"			OFF)
 OPTION(ENABLE_TRACE			"Enables tracing support"				OFF)
 OPTION(LIBGIT2_FILENAME			"Name of the produced binary"				OFF)
-OPTION(USE_SSH				"Link with libssh2 to enable SSH support"		 ON)
+OPTION(USE_SSH				"Enable SSH support. Can be set to a specific backend (libssh2)" ON)
```

I think this needs to be `SET` to properly cache the backend value, like: 

```suggestion
   SET(USE_SSH				"ON" CACHE STRING "Enable SSH support. Can be set to a specific backend (libssh2)")
```